src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -29,51 +29,75 @@ import {
   styleUrls: ['./avatar.component.scss']
 })
 export class SkyAvatarComponent {
-  public get src(): SkyAvatarSrc {
-    return this._src;
-  }
 
+  /**
+   * Indicates whether users can change the image. To select a different image,
+   * users click the image or drag another image on top of it,
+   * much like the `sky-file-drop` component in the
+   * [file attachment module](https://developer.blackbaud.com/skyux/components/fileattachments).
+   */
   @Input()
-  public set src(value: SkyAvatarSrc) {
-    this._src = value;
+  public set canChange(value: boolean) {
+    this._canChange = value;
   }
 
-  public get name(): string {
-    return this._name;
+  public get canChange(): boolean {
+    return this._canChange;
   }
 
+  /**
+   * Specifies the name of the record that the avatar represents.
```

Comment:
In the existing docs `name` is marked as required, but that doesn't make sense given that it's a just a backup value for the `src` property. But just to make sure, we are good with with losing the required label, right?

Also, this property description seems overly long to me. I think we can tighten up to get the point across more clearly.  I'd change the second sentence to `If the src property does not specify an image, the component displays initials from the first and last words in the name.` And I'd combine and tweak the last two sentences: `To ensure that the component extracts the correct initials, specify a name with no prefix or suffix, or just specify initials with a space between them.`

Reply:
Correct, this does not need to be required. Updated text as you suggested.

Reply:
On second glance, you do need to provide either a `src` or a `name`, or the component will render nothing. While each property isn't technically required, you do need one or the other. Should we denote that in the docs somewhere?

Reply:
What do think of adding a sentence to the end of both the `name` and `src` property descriptions to explain the conditional requirement. I'm thinking:
```This property is not required, but the component requires either the `name` or `src` property.```
What do you think?

Reply:
Love it.